src/components/SchemaRow.tsx

```diff
@@ -90,6 +90,7 @@ export const SchemaPropertyRow: typeof SchemaRow = ({ node, onGoToRef, rowOption
 
       <div className="flex-1 flex truncate">
         <Property node={node} onGoToRef={onGoToRef} />
+        {metadata.schema.type && metadata.schema.format && <Format schemaNode={metadata.schema} />}
```

Comment:
and let's do a minor type check, as format can be anything.
```suggestion
        {metadata.schema.type && typeof metadata.schema.format === 'string' && <Format schemaNode={metadata.schema} />}
```